{"name": "test_4[4-5-8]", "status": "failed", "statusDetails": {"message": "AssertionError: assert (4 + 5) == 8", "trace": "self = <testcase.test_calc.TestB object at 0x1106aae48>, a = 4, b = 5, c = 8\n\n    @pytest.mark.parametrize(\"a,b,c\", [\n        (1, 2, 3),\n        (4, 5, 8),\n        (3, 4, 7)\n    ])\n    def test_4(self, a, b, c):\n        logging.info('test_4')\n>       assert a + b == c\nE       assert (4 + 5) == 8\n\ntest_calc.py:56: AssertionError"}, "attachments": [{"name": "log", "source": "a201c751-3b28-4a8c-a9b2-12f4c9ee0cbb-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "b5842b7b-cd9e-4ca4-92dd-b6d4ce804e7f-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "53a13a80-2026-4fbc-b662-c8fe526b1b36-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "a", "value": "4"}, {"name": "b", "value": "5"}, {"name": "c", "value": "8"}], "start": 1564493501597, "stop": 1564493501597, "uuid": "d5720d1b-8646-4509-a05d-174efab76856", "historyId": "94b484c0cfbbf0418119afe1e40c9119", "fullName": "test_calc.TestB#test_4[4-5-8]", "labels": [{"name": "suite", "value": "test_calc"}, {"name": "subSuite", "value": "TestB"}, {"name": "host", "value": "anxiaodongdeMacBook-Pro.local"}, {"name": "thread", "value": "84097-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc"}]}